### DISCUSSION TITLE
Library setting not captured by PnPSiteTemplate

### DISCUSSION THREAD

**Discussion by UserManagedSolutions** [pnp/powershell#2839] [Q&A] [answered]
I am using PnPSiteTemplate to capture the configuration of a single document library, then provision other libraries.
`Get-PnPSiteTemplate -Out template.xml -ListsToExtract "Documents" -Handlers Lists`

That works well for all my settings (fields, views), except for one: the advanced setting to open files in the client application is lost in the process.

**Comment by UserManagedSolutions** [comment]
@veronicageek I see that you have moved this thread from issue to discussion. Does it mean that what I am seeing is the expected behavior, not a bug?
Also, let me clarify that I am using **v1.12**, not 2.0.

**Comment by PaoloPia** [comment, marked as answer]
Hi! Thanks for sharing the feedback. The library property that you are referring to should be _DefaultItemOpenInBrowser_ and as far as I can see, we don't have it in the schema, so for sure we do not extract it. I've just added an [issue](https://github.com/pnp/PnP-Provisioning-Schema/issues/610) in the schema repo, so that we can implement it in the next schema release. Thanks.
